scripts/hail_batch/variant_selection/hgdp_1kg_tob_wgs_variant_selection.py

```diff
@@ -51,10 +51,18 @@ def query(output):  # pylint: disable=too-many-locals
     hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.filter_rows(
         (hl.len(hgdp1kg_tobwgs_joined.alleles) == 2)
         & (hgdp1kg_tobwgs_joined.locus.in_autosome())
-        & (hgdp1kg_tobwgs_joined.variant_qc.AF[1] > 0.001)
+        & (hgdp1kg_tobwgs_joined.variant_qc.AF[1] > 0.01)
         & (hgdp1kg_tobwgs_joined.variant_qc.call_rate > 0.99)
         & (hgdp1kg_tobwgs_joined.IB.f_stat > -0.25)
     )
+
+    hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.cache()
+    nrows = hgdp1kg_tobwgs_joined.count_rows()
+    print(nrows)
+    hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.sample_rows(
+        1000000 / nrows, seed=12345
```

Comment:
Instead of using "magic values" like 1000000 directly in the code, you can make this more readable by using a constant that you pull out to the top, e.g. `NUM_ROWS_BEFORE_LD_PRUNE = 1000000`.

Reply:
added to the top (and I agree, that makes it easier to read).